terraform/user_data.sh

```
@@ -0,0 +1,137 @@
+
+/opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl \
+  -a fetch-config -m ec2 \
+  -c file:/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json -s
+
+systemctl enable amazon-cloudwatch-agent
+
+# --- Pull .env from SSM Parameter Store ---
+APP_DIR="/home/ubuntu/genie_nhs_website"
+
+aws ssm get-parameter \
+  --name "${ssm_env_parameter}" \
```

Shellcheck (0.11.0)

[warning] ssm_env_parameter is referenced but not assigned.

(SC2154)